suite-native/device-manager/src/components/DeviceControlButtons.tsx

```diff
@@ -32,7 +32,7 @@ export const DeviceControlButtons = () => {
     if (!selectedDevice) return null;
 
     const handleEject = () => {
-        dispatch(deviceActions.forgetDevice(selectedDevice));
+        dispatch(deviceActions.deviceDisconnect(selectedDevice));
```

Comment:
We don't want to forget the device on disconnect? IMo it should be forgotted until we do not support remeber mode. Or is the selector just wrongly named and it does something else?

Reply:
6e7f30691